timemachine/cpp/src/kernels/k_neighborlist.cuh

```diff
@@ -11,11 +11,12 @@ void __global__ k_find_block_bounds(
     const int N,                               // Number of atoms
     const int num_tiles,                       // Number of tiles
     const int num_indices,                     // Number of indices
+    unsigned int *ixn_count,                   // [1]
```

Comment:
nit: it seems like elsewhere we've typically ordered kernel arguments with writable pointers at the end. Is there a reason to break this convention here?

Reply:
Moved to end here: [PR Feedback: Follow convention for ordering of params](https://github.com/proteneer/timemachine/pull/1046/commits/e467ad52f35aa030f34331b40355164094056cbb)